### PR TITLE
Switch to picamera2 and add key file

### PR DESCRIPTION
**Pull request by Trotyl15**

### CODE REVIEW

File: blue_only.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+"""
+Auto-landing solution for 2024 competition.
+"""
```
Comment (mgupta27) [inline blue_only.py]: Nit: Change to `Auto-landing script for AEAC 2024 competition.`

File: blue_only.py [hunk 2]
```diff
@@ -6,16 +9,21 @@
 
 import cv2
 import numpy as np
+from dotenv import load_dotenv
```
Comment (mgupta27) [inline blue_only.py:13]: Nit: Add a space above.

File: requirements.txt [hunk 3]
```diff
@@ -3,4 +3,5 @@ lxml==5.1.0
 numpy==1.24.4
 opencv-python==4.9.0.80
 pymavlink==2.4.41
```
Comment (mgupta27) [inline requirements.txt:5]: Out of curiosity, do we need to have a particular version for these libraries?

**Review by mgupta27 — APPROVED** [review]
LGTM! Just a few nits.

File: requirements.txt (hunk 3 above)
Comment (Trotyl15) [reply requirements.txt:5]: just incase they have conflicts

**Pull request merged**